src/WebJobs.Extensions.DurableTask/IPlatformInformation.cs

```diff
@@ -6,45 +6,74 @@
 namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
 {
     /// <summary>
-    /// Interface for accessing the AppService plan information,
-    /// the OS, and user-facing PL.
-    ///
-    /// Note: The functionality is currently limited, but will grow
-    /// along with the pursuit of more platform-specific defaults.
+    /// Representation of the supported Operating Systems.
     /// </summary>
-    [Obsolete]
-    public interface IPlatformInformationService
+    public enum OperatingSystem
+    {
+        /// <summary>
+        /// Linux OS.
+        /// </summary>
+        Linux,
+
+        /// <summary>
+        /// Windows OS
+        /// </summary>
+        Windows,
+    }
+
+    /// <summary>
+    /// Representation of supported Programming Languages.
+    /// </summary>
+    public enum WorkerRuntimeType
     {
         /// <summary>
-        /// Determines if the application is running on a Consumption plan,
-        /// irrespective of OS.
+        /// C-Sharp.
         /// </summary>
-        /// <returns>True if running in Consumption. Otherwise, False.</returns>
-        bool InConsumption();
+        Csharp,
```

Comment:
rename from `Csharp` to `DotNetInProcess` for the eventual support for the out of process .NET worker.